functions/src/functions/webhooks/index.ts

```diff
@@ -19,3 +20,6 @@ export const webhookCreateAllSurveys = createAllSurveysFunction;
 export const webhookStripeCharge = stripeWebhookFunction;
 
 export const webhookCreateDonationCertificates = createDonationCertificatesFunction;
+
+// Twilio OTP verification function
+export const webhookTwilioVerify = twilioVerifyFunction;
```

Comment:
Maybe call this `webhookTwilioOtpVerification`? Or something similar that makes clear what is verified here.

Reply:
Applied in new PR https://github.com/socialincome-san/public/pull/1119/files